src/azure-cli/azure/cli/command_modules/sql/custom.py

```diff
@@ -1980,10 +1979,8 @@ def encryption_protector_update(
     return client.create_or_update(
         resource_group_name=resource_group_name,
         server_name=server_name,
-        parameters=EncryptionProtector(
-            server_key_type=server_key_type,
-            server_key_name=key_name
-        )
+        server_key_type=server_key_type,
+        server_key_name=key_name
```

Comment:
Why is this changed?

Reply:
Build wasn't successful because of failed test **test_sql_tdebyok (azure.cli.command_modules.sql.tests.latest.test_sql_commands.SqlTransparentDataEncryptionScenarioTest)**. You can find more on this link https://github.com/Azure/azure-cli/runs/214404678. 
After I've fixed this, all checks have passed

Reply:
https://github.com/Azure/azure-sdk-for-python/commit/58857aeff4ffc4edfe383f2f605bf10fd42f2b7c

@zikalino , can you clarify why this breaking change happened?